### PR TITLE
Format `.js` and `.scss` files using Prettier

### PR DESCRIPTION
**Pull request by dnkrj**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

# What
- Introduce and configure prettier
- Update files to reflect new formatting rules

# Why
- Align with govuk-frontend

https://trello.com/c/p8IqI4zH/1653-tech-spike-more-opinionated-es6-rules

### CODE REVIEW

File: app/assets/javascripts/admin/analytics.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(function (GOVUK) {
+;(function (GOVUK) {
```
Comment (ryanb-gds) [inline app/assets/javascripts/admin/analytics.js:1]: This looks weird. Wonder what rule is causing that?

File: app/assets/stylesheets/vendor/chosen-bootstrap.scss
Comment (ryanb-gds) [inline app/assets/stylesheets/vendor/chosen-bootstrap.scss:1]: Should this be ignored?

**Review by ryanb-gds — COMMENTED** [review]
Couple of questions. The leading semi-colons one seems a bit odd, never seen that before

File: app/assets/stylesheets/vendor/chosen-bootstrap.scss
Comment (dnkrj) [reply app/assets/stylesheets/vendor/chosen-bootstrap.scss:1]: Good point! Have added `vendor` and `admin_legacy` to `.prettierignore`

File: app/assets/javascripts/admin/analytics.js (hunk 1 above)
Comment (dnkrj) [reply app/assets/javascripts/admin/analytics.js:1]: Ah, thanks for finding that!

**Review by ryanb-gds — APPROVED** [review]
Good to go. Very pretty

**Pull request merged**